assets/messages.yml

```diff
@@ -417,12 +398,12 @@ messages:
       shortcut: dup
       category: duplicate
       message: |-
-        *Thank you for your report!*
-        We're tracking this issue in *%s%*, so this ticket is being resolved and linked as a *duplicate*.
+        **Thank you for your report!**
+        We're tracking this issue in **%s%**, so this ticket is being resolved and linked as a **duplicate**.
 
         If you would like to add a vote and any extra information to the main ticket it would be appreciated.
 
-        If you haven't already, you might like to make use of the [*+search feature+*|https://bugs.mojang.com/issues/?jql=project=%project_id%] to see if the issue has already been mentioned.
+        If you haven't already, you might like to make use of the [**search feature**](https://bugs.mojang.com/) to see if the issue has already been mentioned.
```

Comment:
I think I would prefer for this link to be `https://report.bugs.mojang.com/servicedesk/customer/portal/%project_id%`.

You will need to change the project id variables at the top as follows for this to work properly (the project id is only used in these links afaict)

```yml
  project_id:
    - project: mc
      value: '2'
    - project: mcd
      value: '3'
    - project: mcpe
      value: '6'
    - project: mcl
      value: '7'
    - project: mclg
      value: '8'
    - project: bds
      value: '4'
    - project: realms
      value: '9'
    - project: web
      value: '10'
```

Reply:
Have made changes